### PR TITLE
Makes camp fire kit make campfires via alt click rather then crafting

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## Description
Alt click a fire kit to make a camp fire!

## Motivation and Context
Less crafting menu bloat

## How Has This Been Tested?
Its not, likely dosnt work

## Screenshots (if appropriate):
Its not needed
## Changelog (necessary)
:cl:
tweak: Camp fire kits now deploy into a camp fire rather then crafting into one. How novel 
tweak: Also, raises the crafting time of the camp fire kit to 120 from 80 (4 more seconds)
/:cl:

### CODE REVIEW

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -60,14 +60,7 @@
...
 	reqs = list(/obj/item/stack/sheet/mineral/wood = 15)
-	time = 80
-	category = CAT_MISC
-
-/datum/crafting_recipe/campfirekit
-	name = "set up campfire kit"
-	result = /obj/structure/campfire
-	reqs = list(/obj/item/crafting/campfirekit = 1)
-	time = 40
+	time = 120 // Around 12 seconds
```
Comment (MarinaGryphon) [inline code/modules/crafting/recipes.dm:63]:
```suggestion
	time = 12 SECONDS
```
We have a macro for this.

File: code/modules/fallout/obj/crafting.dm [hunk 2]
```diff
@@ -168,5 +168,11 @@
...
-	icon_state = "lunchbox"
\ No newline at end of file
+	desc = "A small box filled with an assortment of wood and tender. Useful for quickly making a fire. Alt click to crate a camping fire."
+	icon_state = "lunchbox"
+
+/obj/item/crafting/campfirekit/AltClick(mob/user)
+	. = ..()
+	var/turf/T = get_turf(usr)
+	new /obj/structure/campfire(T)
+	return ..()
```
Comment (MarinaGryphon) [inline code/modules/fallout/obj/crafting.dm:178]:
```suggestion
```
You're double-calling it. Either call it with `. = ..()` at the start or `return ..()` at the end, but they mean the same thing.

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/crafting/recipes.dm:63]: No other crafting item to me is like that

**Comment by judgex** [comment]
I'm not merging something that the author says "likely doesn't work".

Get back to me with some confirmation of function and I will merge this.

**Pull request closed without merging**